### PR TITLE
Add configuration for login token expiry

### PR DESCRIPTION
**Pull request by guy-har**

### CODE REVIEW

**Review by nopcoder — CHANGES_REQUESTED** [review]
Looks good, just call the configuration 'login duration'

File: clients/java/src/test/java/io/lakefs/clients/api/model/AuthenticationTokenTest.java [hunk 1]
```diff
@@ -48,4 +48,12 @@ public void tokenTest() {
         // TODO: test token
     }
 
+    /**
+     * Test the property 'tokenExpiration'
+     */
+    @Test
+    public void tokenExpirationTest() {
```
Comment (idanovo) [inline clients/java/src/test/java/io/lakefs/clients/api/model/AuthenticationTokenTest.java:55]: can you please open a ticket for `AuthenticationTokenTest` tests? we have a lot of TODOs here

**Review by idanovo — APPROVED** [review]

File: clients/java/src/test/java/io/lakefs/clients/api/model/AuthenticationTokenTest.java (hunk 1 above)
Comment (guy-har) [reply clients/java/src/test/java/io/lakefs/clients/api/model/AuthenticationTokenTest.java:55]: @idanovo I'm not sure that we currently want to implement the tests for our generated clients

**Review by nopcoder — CHANGES_REQUESTED** [review]
Code looks good - can you add a test in controller_test.go to verify login works and you are getting the right value?

**Review by ortz — APPROVED** [review]

**Review by nopcoder — APPROVED** [review]
LGTM

**Pull request merged**